src/main/java/thePackmaster/cards/creativitypack/Mimicry.java

```diff
@@ -33,13 +33,7 @@ public void use(AbstractPlayer abstractPlayer, AbstractMonster abstractMonster)
         addToBot(new AbstractGameAction() {
             @Override
             public void update() {
-                ArrayList<AbstractCard> list = new ArrayList<>();
-                for (AbstractCardPack pack : SpireAnniversary5Mod.currentPoolPacks) {
-                    if (!CreativityPack.ID.equals(pack.packID)) {
-                        list.addAll(pack.cards);
-                    }
-                }
-                list.removeIf(c -> c.rarity != CardRarity.COMMON);
+                ArrayList<AbstractCard> list = Wiz.getCardsMatchingPredicate(c -> c.rarity == CardRarity.COMMON && !CreativityPack.ID.equals(SpireAnniversary5Mod.cardParentMap.getOrDefault(c.cardID, null)) && !c.hasTag(CardTags.HEALING));
```

Comment:
This code was didn't have the check to exclude healing tagged cards previously, which nobody noticed because there probably aren't any common healing cards.